packages/cyberstorm/src/components/Forms/CreateTeamForm/CreateTeamForm.tsx

```diff
@@ -0,0 +1,123 @@
+"use client";
+import { faArrowsRotate } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { useController, useForm, Form } from "react-hook-form";
+import * as Button from "../../Button";
+import { TextInput } from "../../TextInput/TextInput";
+import { useToast } from "../../Toast/Provider";
+import styles from "./CreateTeamForm.module.css";
+import { z } from "zod";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { isErrorResponse } from "../../../utils/type_guards";
+import { useEffect } from "react";
+
+export function CreateTeamForm() {
+  const toast = useToast();
+
+  // "Required field" error is different from value being too little
```

Comment:
This comment raises more questions than answers. How is it different? What do we need required_error for if we have a separate check for value length?